src/dex/kyberdmm/fee-formula.ts

```diff
@@ -62,8 +72,8 @@ type TrendData = {
   lastTradeBlock: bigint;
 };
 
-const SHORT_ALPHA = (BigInt(2) * PRECISION) / BigInt(5401);
-const LONG_ALPHA = (BigInt(2) * PRECISION) / BigInt(10801);
+const SHORT_ALPHA = (BI_2 * PRECISION) / BigInt(5401);
```

Comment:
`BigInt(5401)` and `BigInt(10801)` on the next line should be replaced with constants, otherwise we recreate them on every function call

Reply:
Replaced with `n` postfixed